src/main/java/frc/robot/subsystems/Indexer.java

```diff
@@ -67,6 +32,22 @@ public double getWheelVelocity() {
     return 0.0;
   }
 
+  public void isCurrentCargoColored(){
```

Comment:
change to isCurrentCargoRed (apparently FMS tells us if our alliance color is red or not)